docs/development.rst

```diff
@@ -146,7 +146,7 @@ as an example) and run migrations:
 
 .. code-block:: bash
 
-    $ paver update_db
+    $ make migrate
     # Or use a more down-to-the-root command (replace aws with your version of config)
```

Comment:
Not part of your PR, but what is meant by `aws` here?

Reply:
Also, I just remembered that the old call had `paver install_prereqs` baked in. Does that need to be added here?

Reply:
We have `make requirements`, but I don't think this documentation requires the requirements to be installed. (`aws` is the old name of the `production` settings, I believe.)

Reply:
1. This comment doesn't make sense in this file, because we're using `devstack`, and it probably doesn't need to be replaced?
2. In the other file, it probably makes sense to replace `aws` in the comment and the example command to `production`.

Since this merged already, I guess we'll see if one of us do a follow-up PR. :)